qiskit_experiments/framework/experiment_data.py

```diff
@@ -67,6 +68,23 @@ def completion_times(self) -> Dict[str, datetime]:
 
         return job_times
 
+    @classmethod
+    def load(cls, experiment_id: str, service: DatabaseServiceV1) -> "ExperimentData":
+        """Load a saved experiment data from a database service.
+
+        Args:
+            experiment_id: Experiment ID.
+            service: the database service.
+
+        Returns:
+            The loaded experiment data.
+        """
+        db_expdata = DbExperimentDataV1.load(experiment_id, service=service)
+        expdata = ExperimentData()
```

Comment:
What happens if we load composite one? I think user should be able to define own `ExperimentData` subclass and use it for analysis with convenient methods or data structure. This means this data conversion will not cover all situation we should assume.

Reply:
You would need to implement this method in every subclass to return the correct type. But currently composite experiments can't be loaded anyway, which is what Yael's PR is for

Reply:
> every subclass to return the correct type

What do you mean by this? We need to implement a subclass of `DbExperimentDataV1` rather than `ExperimentData`? Or override `load` method in each subclass? If you meant latter option, I think this typecasting should be done in `ExperimentData` rather than `DbExperimentDataV1`.